### PR TITLE
Fixes UTM input bug

### PR DESCRIPTION
**Pull request by seamus-oconnor**
https://apps.nrs.gov.bc.ca/int/jira/browse/WATER-482

Replaces Coords.vue input locking & watches with on-input coords conversion logic

### CODE REVIEW

File: app/frontend/src/submissions/components/SubmissionForm/Coords.vue [hunk 1]
```diff
@@ -508,8 +340,50 @@ export default {
         zone: null
       }
     },
+    handleDegreesChange () {
```
Comment (dainetrinidad) [inline app/frontend/src/submissions/components/SubmissionForm/Coords.vue]: Would be nice if there were unit tests for these new methods

Comment (seamus-oconnor) [reply app/frontend/src/submissions/components/SubmissionForm/Coords.vue]: They are not new - just changed from the watch-ers style from before. I will try and add some unit tests to it though.

Comment (seamus-oconnor) [reply app/frontend/src/submissions/components/SubmissionForm/Coords.vue]: @dainetrinidad I have added tests to the handle*Change methods.

Comment (dainetrinidad) [reply app/frontend/src/submissions/components/SubmissionForm/Coords.vue]: Thank you!

**Pull request merged**